lldb/source/Plugins/ObjectFile/PECOFF/WindowsMiniDump.cpp

```diff
@@ -21,11 +21,12 @@
 namespace lldb_private {
 
 bool SaveMiniDump(const lldb::ProcessSP &process_sp,
-                  const lldb_private::FileSpec &outfile,
+                  const SaveCoreOptions &core_options,
                   lldb_private::Status &error) {
   if (!process_sp)
     return false;
 #ifdef _WIN32
+  const auto &outfile = core_options.GetOutputFile().value();
```

Comment:
1. This is unused but I assume it will be used later?
2. This warning is produced in a clang-cl build on Windows:
```
C:\Users\tcwg\llvm-worker\lldb-aarch64-windows\llvm-project\lldb\source\Plugins\ObjectFile\PECOFF\WindowsMiniDump.cpp(29,25): warning: object backing the pointer will be destroyed at the end of the full-expression [-Wdangling-gsl]
```